src/app.rs

```diff
@@ -175,6 +177,42 @@ pub struct Config {
     pub using_table: Option<String>,
     pub using_port: Option<u32>,
     // pub cache_expiration_time: Option<i64>, // in second. default 300 (= 5 minutes)
+    #[serde(default)]
+    pub retry_batch_write_item: RetryConfig,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct RetryConfig {
+    pub enabled: bool,
+    #[serde(default)]
+    pub jitter: bool,
+    pub factor: Option<f32>,
+    pub min_delay: Option<u64>,
+    pub max_delay: Option<u64>,
+    pub max_times: Option<usize>,
+}
+
+impl From<RetryConfig> for ExponentialBuilder {
+    fn from(value: RetryConfig) -> Self {
```

Comment:
Please introduce validation for the input if possible. However, you can leave it as follow-up work in this pull request.

Reply:
What validation do I need to put in? Here we are just passing the contents of the `RetrySetting` as is to the ExponentialBuilder, and we are aware that incorrect formatting, etc. will result in an error when deserializing the config file.

Reply:
For example, `max_attempts` is greater than zero because [it includes the initial attempt](https://docs.rs/aws-sdk-dynamodb/latest/aws_sdk_dynamodb/config/retry/struct.RetryConfig.html#method.with_max_attempts).

> Set the maximum number of times a request should be tried, including the initial attempt. This value must be greater than zero.

Also, `max_backoff` should be greater than zero. But it is not required.

Reply:
I add the validation.